bot.py

```diff
@@ -82,19 +86,13 @@ def get_subscriptions(self):
         self.send(subscriptions_msg)
 
     def subscribe(self, channel, channel_id):
-        subscribe_msg = {
-            "msg": "sub",
-            "id": channel,
-            "name": "stream-room-messages",
-            "params":[
-                channel_id,
-                False
-            ]
-        }
+        subscribe_msg = {"msg": "sub", "id": channel, "name": "stream-room-messages", "params": [channel_id, False]}
         self.send(subscribe_msg)
 
     def format_issue(self, repository, issue):
-        headers = { 'User-Agent': 'Godot Issuebot by hpvb', }
+        headers = {
+            "User-Agent": "Godot Issuebot by hpvb",
+        }
```

Comment:
I'm also not sure why this ends up on two lines when larger structures end up on a single line.

Reply:
This one is because of the trailing comma. If we remove the trailing comma, it will become one line again.

Reply:
Would you mind making those changes then so I don't have to merge this large diff and then create another large one right away to correct this?

Reply:
Done